src/test/java/io/rocketpartners/cloud/action/sql/SqlServiceFactory.java

```diff
@@ -59,31 +59,13 @@ public static void main(String[] args) throws Exception
 
    }
 
-   public static synchronized Service service() throws Exception
+   public static synchronized Service service(String type) throws Exception
```

Comment:
We don't want to pass an arg here. we want to sql service to configure both the /h2 and /mysql paths every time.

Reply:
Actually, don't change this yet. I want to figure out how to transparently run all the sql test for all db types

Reply:
So we need the same service to support multiple DBs same time, right? so that path '/h2' will access h2 DB, '/mysql' will access mysql DB.